src/main/java/org/geppetto/frontend/controllers/ConnectionHandler.java

```diff
@@ -190,6 +193,10 @@ public void loadGeppettoProject(String requestID, IGeppettoProject geppettoProje
 			setConnectionProject(geppettoProject);
 			websocketConnection.sendMessage(requestID, OutboundMessages.PROJECT_LOADED, projectJSON);
 
+			boolean persisted = !geppettoProject.isVolatile();
+			String update = "{\"persisted\":" + persisted + ",\"projectId\":" + geppettoProject.getId()  +"}";
```

Comment:
How about doing it in a way where it goes through the Message Template?  https://github.com/openworm/org.geppetto.frontend/blob/master/src/main/webapp/js/websocket-handlers/GEPPETTO.MessageSocket.js#L216